### DISCUSSION TITLE
minio client HealthCheck functionality question.

### DISCUSSION THREAD

**Discussion by szabokacsa** [minio/minio#14228] [Q&A] [answered]
Hi,
I tried the HealthCheck funtionality of minio client. I use this test as an example: [code](https://github.com/minio/minio-go/blob/master/healthcheck_test.go)

I did not start the minio server, but got IsOnline() == true response after I started the HealthCheck.

My code contains the following steps.
1. start HeathCheck
2. check IsOnline response
3. if minio is online call other minio functions
4. if minio is offline, return error

Am I useing the healthCheck funtionality the correct way?

Thanks for your response!

**Comment by j-heaton** [comment, marked as answer]
request this url instead

`<your_minio_endpoint_>:<port>/minio/health/live`

A response code of 200 OK indicates the MinIO server is online and functional. Any other HTTP codes indicate an issue with reaching the server, such as a transient network issue or potential downtime.

more info here: https://min.io/docs/minio/linux/operations/monitoring/healthcheck-probe.html
